locale/en.yml

```
@@ -1480,6 +1481,7 @@ en:
       ManageIQ::Providers::EmbeddedAnsible::AutomationManager::VmwareCredential:          Credential (VMware)
       ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook:                  Playbook (Embedded Ansible)
       ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource: Repository (Embedded Ansible)
+      ManageIQ::Providers::ExternalAutomationManager:                                     Automation Manager
```

@kavyanekkalapu not sure if this is what you meant, please review

---

Think it's supposed to be plural

```suggestion
      ManageIQ::Providers::ExternalAutomationManager:                                     Automation Managers
```

---

NOTE as for sorting, I put this after the `ManageIQ::Providers::EmbeddedAnsible::AutomationManager` entries even though `ManageIQ::Providers::AutomationManager` is next :shrug: not sure if we want to reorg this whole list right now

---

Actually maybe not...I see others as signular too...will let @kavyanekkalapu review :)

---

We show this for table title. so i am assuming it is plural.

---

Actually nvm i got confused too like everyone. let me double check on singular and plural.

---

What else do we show as a table title?
```
ExtManagementSystem:            Provider
ManageIQ::Providers::InfraManager:                                                  Infrastructure Provider
ManageIQ::Providers::CloudManager:                                                  Cloud Provider
```
I'm guessing being made plural is handled in a different place given all of these examples

---

Also, should we name this `Automation Provider` instead of `Automation Manager` to be consistent with the infra/cloud/container/etc... ones?

---

@agrare @Fryguy It is working with singular.